tests/bgp/bgp_helpers.py

```diff
@@ -118,3 +135,35 @@ def verify_all_routes_announce_to_bgpmon(duthost, ptfhost):
         if route not in bgpmon_routes:
             return False
     return True
+
+def remove_bgp_neighbors(duthost, asic_index):
+    """
+    Remove the bgp neigbors for a particular BGP instance
+    """
+    namespace = duthost.get_namespace_from_asic_id(asic_index)
+
+    # Convert the json formatted result of sonic-cfggen into bgp_neighbors dict
+    bgp_neighbors = json.loads(duthost.command("sudo sonic-cfggen {} -d --var-json {}".format('-n ' + namespace if namespace else '', "BGP_NEIGHBOR"))["stdout"])
+    cmds = []
+    for neighbor in bgp_neighbors.keys():
+        cmds.append('sudo sonic-db-cli {} CONFIG_DB del "BGP_NEIGHBOR|{}"'.format('-n ' + namespace if namespace else '', neighbor))
+    duthost.shell_cmds(cmds=cmds)
```

Comment:
we can remove loop and sonic-cfggen call.
this seems to work:
sonic-db-cli CONFIG_DB keys "BGP_NEI*" | xargs  sonic-db-cli CONFIG_DB del

Reply:
Updated.